containers/fedora/build.sh

```
@@ -71,22 +89,112 @@ if [ $FEDORA_VERSION -gt 40 ]; then
+        mount /dev/sda2 / : \
+        download "/loader/entries/${BLS_ENTRY}" "${BLS_ENTRY_TMP}"
+    sed -i 's/^\(options .*\)/\1 systemd.default_device_timeout_sec=300/' "${BLS_ENTRY_TMP}"
+    guestfish -a "${WORK_IMAGE}" \
+        run : \
+        mount /dev/sda2 / : \
+        upload "${BLS_ENTRY_TMP}" "/loader/entries/${BLS_ENTRY}"
+    rm -f "${BLS_ENTRY_TMP}"
+fi
+
+# Clean up any leftovers from a previous failed run
+rm -rf $BUILD_DIR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)